models/netsuite2/src_netsuite2.yml

```diff
@@ -208,6 +208,8 @@ sources:
             description: The current rate associated with the exchange rate.
           - name: averagerate
             description: The consolidated exchange rates average rate.
+          - name: accountingbook
+            description: Reference to the accounting book of the entry.
```

Comment:
There is already an [accounting book id](https://github.com/fivetran/dbt_netsuite_source/blob/546cc57fd380dd20f73ce0027a1009ad58836b53/models/stg_netsuite2.yml#L33-L34) docs.md that we can reference here. Can you update this to leverage the docs file. This way we maintenance efforts can be minimized.

Reply:
Sure thing!